frame/democracy/src/types.rs

```diff
@@ -204,3 +205,17 @@ pub enum UnvoteScope {
 	/// Permitted to do only the changes that do not need the owner's permission.
 	OnlyExpired,
 }
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct Deposit<AccountId, Balance> {
+	pub(crate) who: AccountId,
+	pub(crate) amount: Balance,
+}
+
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[codec(mel_bound(Deposit: MaxEncodedLen, BoundedVec<u8, Limit>: MaxEncodedLen))]
+#[scale_info(skip_type_params(Limit))]
+pub struct Metadata<Deposit, Limit: Get<u32>> {
```

Comment:
Should the metadata also be added for the Tips, Bounties and Collective pallets, since they are also go through voting?